templates/CDK-full-template.yaml

```diff
@@ -57,16 +64,27 @@ Resources:
         - Key: Name
           Value: VPC
 
-  PublicSubnet:
+  PublicSubnetTwo:
     Type: AWS::EC2::Subnet
     Properties:
-      CidrBlock: !FindInMap ['SubnetConfig', 'Public', 'CIDR']
+      CidrBlock: !FindInMap ['SubnetConfig', 'PublicTwo', 'CIDR']
       VpcId: !Ref VPC
       AvailabilityZone: !Ref Region
       MapPublicIpOnLaunch: true
       Tags:
        - Key: Name
-         Value: PublicSubnet
+         Value: PublicSubnetTwo
+  
+  PublicSubnetOne:
+    Type: AWS::EC2::Subnet
+    Properties:
+      CidrBlock: !FindInMap ['SubnetConfig', 'PublicOne', 'CIDR']
+      VpcId: !Ref VPC
+      AvailabilityZone: eu-west-1a
```

Comment:
Should we make this a parameter as well? If I am following the deployment guide and deploying in a different region, today I am only prompted for the `Region` (default value = eu-west-1c).

To ensure a user can change this when going through the steps, I feel this should also be paramaterized?

Reply:
I agree too, to simplify use the same input Region for both subnets, this would be simpler to understand from user point of view

Reply:
Tried this but got an error on deployment: "Resource handler returned message: "The DB subnet group doesn't meet Availability Zone (AZ) coverage requirement."

Reply:
yes, will do this in the new commit!